install.sh

```diff
@@ -382,6 +384,34 @@ get_default_install_dir() {
     [ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.temporalio" || printf %s "${XDG_CONFIG_HOME}/temporalio"
 }
 
+get_install_dir() {
+    local _dir
+    _dir="$(get_default_install_dir)"
+    if [ -n "${TEMPORAL_DIR-}" ]; then
+        _dir="$TEMPORAL_DIR"
+    fi
+    printf %s "$_dir"
+}
+
+install_env() {
+    local _dir="$1"
+
+    cat >"$_dir/env" <<EOL
+#!/bin/sh
+case ":\${PATH}:" in
+    *:"$_dir":*)
+        ;;
+    *)
+        export PATH="$_dir:\$PATH"
+        ;;
+esac
+EOL
+
+    local _source=". $_dir/env"
+    say "To add temporal to PATH update your user profile with the following (ex. ~/.bash_profile):"
```

Comment:
I wouldn't create an env file, just suggest they add the directory to their path, I think we can expect users to be able to do that.

Reply:
This still could be helpful especially for students / people who just learn to code?

Reply:
Can do, the env file won't work with all shells though so I'd at least want to make that clear, "If you are using bash you can..."

Reply:
resolving, started this discussion